README.md

```diff
@@ -1259,6 +1259,54 @@ raw_index = twitter_raw
 enriched_index = twitter_enriched
 api-token = XXXX
 ```
+
+## Troubleshooting
+
+Following is a list of common problems encountered while setting up Grimoire Lab
+
+---
+**NOTE**
+
+You have to run ```docker-compose up``` without the ```-d``` or ```--detach``` option while starting/(re)creating/building/attaching containers for a service. 
```

Comment:
I'm not sure if this part is needed. Why you want to include it?

Reply:
In the "Source code and docker section" the command mentioned is ```docker-compose up -d```. This will run the containers in detached mode which would prevent them from seeing the log. I thought most of the new developers unacquainted to docker might not know this. But the final decision is yours :).

Reply:
Ok, I see. 

Can you include this info in the text above, for instance:
```
Run ```docker-compose up``` without the ```-d``` or ```--detach``` option to see the logs while starting/(re)creating/building/attaching containers for a service. 
```

```
In order to see the logs, run ```docker-compose up``` without the ```-d``` or ```--detach``` option while starting/(re)creating/building/attaching containers for a service. 
```

WDYT?

Reply:
Yours is better as usual :). On it.